src/adapters/github.js

```diff
@@ -350,24 +350,51 @@ class GitHub extends PjaxAdapter {
       cfg.headers = {Authorization: 'token ' + opts.token};
     }
 
-    $.ajax(cfg)
-      .done((data) => {
-        if (path && path.indexOf('/git/trees') === 0 && data.truncated) {
-          const hugeRepos = this.store.get(STORE.HUGE_REPOS);
-          const repo = `${opts.repo.username}/${opts.repo.reponame}`;
-          const repos = Object.keys(hugeRepos);
-          if (!hugeRepos[repo]) {
-            // If there are too many repos memoized, delete the oldest one
-            if (repos.length >= GH_MAX_HUGE_REPOS_SIZE) {
-              const oldestRepo = repos.reduce((min, p) => (hugeRepos[p] < hugeRepos[min] ? p : min));
-              delete hugeRepos[oldestRepo];
-            }
-            hugeRepos[repo] = new Date().getTime();
-            this.store.set(STORE.HUGE_REPOS, hugeRepos);
-          }
-          this._handleError(cfg, {status: 206}, cb);
-        } else cb(null, data);
+    const CACHE_NAME = 'cached-repo-v1';
+    const STORE_NAME = 'cached-repo-time';
+    const CACHE_KEY = url;
+    if(location.hash.search('frc=') !== -1){
+        force = true;
+    }
+    caches.open(CACHE_NAME).then(cache => {
+      cache.match(CACHE_KEY).then(currentMatch => {
+        if(typeof currentMatch === "undefined" || force){
```

Comment:
Please follow the coding convention with spaces.

`if (`, `) {`.

Reply:
I read the contribution section in the README, however, not clear how to follow. Do you guys use Prettier or something? How do I know what style that the code is using? Or is it just obvious by looking at the code?
Sorry for the silly questions, this is my first PR, I'm not usually contribute to projects on Github.